### PR TITLE
Merge framework into Master

### PR DESCRIPTION
**Pull request by 7imbrook**
Restarting the Master branch with the new typing framework. mergeFramwork has all branches in it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,23 @@
-Accessbraille
-=============
+# DRAFT
```
Comment (7imbrook) [inline README.md:1]: We need to rewrite the Readme so its not a draft anymore

File: AccessBraille/AccessBraille/menuDiscriptions.plist [hunk 2]
```diff
@@ -6,5 +6,6 @@
 	<string>The Home menu is where you can always get back to learn about all the features of AccessBraille.</string>
 	<string>In Settings you can confgure AccessBraille to work best for you.</string>
 	<string>Get to know AccessBraille and how to get around.</string>
+	<string>This is the flash card discription, it needs one...</string>
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/menuDiscriptions.plist:9]: Will need full description

File: AccessBraille/AccessBraille/TextOut.m [hunk 3]
```diff
@@ -156,17 +223,30 @@ - (void)rewrite {
     }
 }
 
+/**
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextOut.m:226]: Doesn't seem connected

File: AccessBraille/AccessBraille/TextOut.m [hunk 4]
```diff
@@ -29,73 +32,134 @@ - (id)initWithFrame:(CGRect)frame {
...
     
-    // Clear Text
+    /// Clear Text within Display Rectangle
     clearText = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(clearText)];
     [clearText setNumberOfTouchesRequired:3];
     [self addGestureRecognizer:clearText];
     
-    
-    // Style
+    /// Style
```
Comment (piperchester) [inline AccessBraille/AccessBraille/TextOut.m:61]: Improper Commenting

File: AccessBraille/AccessBraille/TextOut.m [hunk 5]
```diff
@@ -29,73 +32,134 @@ - (id)initWithFrame:(CGRect)frame {
...
     }
-    [chars addObject:buildWord];
-    buildWord = @"";
+    
+    [chars addObject:nextWord];
+    nextWord = @"";
     return chars;
 }
 
+/**
```
Comment (7imbrook) [inline AccessBraille/AccessBraille/TextOut.m:121]: Please Remove

File: AccessBraille/AccessBraille/TextOut.m [hunk 6]
```diff
@@ -29,73 +32,134 @@ - (id)initWithFrame:(CGRect)frame {
...
         return false;
     }
+    
     NSMutableArray *chars = [[NSMutableArray alloc] init];
-    NSString *buildWord = @"";
-    for(int index = 0; index<=[str length]-1; index++){
-        if ([str characterAtIndex:index] == 32){
-            [chars addObject:buildWord];
-            buildWord = @"";
+    NSString *nextWord = @"";
```
Comment (piperchester) [inline AccessBraille/AccessBraille/TextOut.m:102]: Remove cursor implemenation

**Comment by 7imbrook** [comment]
Fix Comments, then resubmit pull request

**Pull request closed without merging**